in_progress/roles.md

```diff
@@ -0,0 +1,91 @@
+
+# Roles And Responsibilities
+
+## The Community
+
+The scverse community consists of anyone using or working with the project in any way.
+
+## Contributors/ Project developers
+
+A community member can become a contributor by interacting directly with the project in concrete ways, such as:
+
+- proposing, discussing, or reviewing a change to the code, documentation, or specification via a GitHub pull request to any https://github.com/scverse repository;
+- reporting a GitHub issue to any https://github.com/scverse repository;
+- discussing examples or usage issues on any of our discussion platforms
+
+among other possibilities.
+Any community member can become a contributor, and all are encouraged to do so.
+By contributing to the project, community members can directly help to shape its future.
+
+All community members are required to adhere by our code of conduct (**TODO: add link**).
+Potential contributors are encouraged to read the Contributing Guide (**TODO: add link**).
+
+A community member becomes a contributor when the following criteria are met:
+
+- At least two core developers from separate employers agree membership is a good idea.
+- The new member has supported the project several times, either through code or otherwise, and has demonstrated a willingness to help solve other people's problems, not just their own.
+
+## Core developers
+
+Core developers are community members that have demonstrated continued commitment to the project through ongoing contributions.
+They have shown they can be trusted to maintain the scverse with care.
+Becoming a core developer allows contributors to merge approved pull requests, cast votes for and against merging a pull-request, and be involved in deciding major changes to the API, and thereby more easily carry on with their project related activities. 
+Core developers appear as organization members on the scverse [GitHub organization](https://github.com/orgs/scverse/people) and are on our [@scverse/core-devs](https://github.com/orgs/scverse/teams/core-devs) GitHub team (**TODO**).
+Core developers are expected to review code contributions while adhering to the [core developer guide](CORE_DEV_GUIDE.md) (**TODO**).
+New core developers can be nominated by any existing core developer, and for details on that process see our core developer guide.
+All core packages must have at least one core developer as a maintainer.
+Primary maintainers of new core packages, who are not already core developers, will be invited to join.
+
+## Steering Council
+
+The Steering Council (SC) members are core developers who have additional responsibilities to ensure the smooth running of the project.
+SC members are expected to participate in strategic planning, approve changes to the governance model, and make decisions about funding granted to the project itself.
+(Funding to community members is theirs to pursue and manage). 
```

Comment:
this not sure, I think core devs can also apply and manage to funding, but prbably legally this would ahve to happen via a SC